contributors/devel/pull-requests.md

```diff
@@ -94,7 +100,7 @@ Release notes apply to PRs on the master branch. For cherry-pick PRs, see the [c
 **Comments**
 
 - If the PR does not need to be mentioned at release time, leave the `/release-note-none` comment command.
```

Comment:
Since there is only one comment that is accepted you could get rid of the "Descriptions" and "Comments" sections and just say that a `/release-note-note` comment can still be used as an alternative to writing "NONE" in the release-note block if the block is left empty.